regression_tests/ref_counter.jl

```diff
@@ -1 +1 @@
-15
+16
```

Comment:
We need to reset all the mse values to zero when the reference counter is incremented.